doc-content/jbpm-docs/src/main/asciidoc/Persistence/persistence-configuration-con.adoc

```diff
@@ -0,0 +1,8 @@
+[id='persistence-configuration-con_{context}']
+= Configuration of persistence in the {PROCESS_ENGINE}
```

Comment:
All of the sub-modules in this section need to be either converted to procs or re-titled as proper cons (noun phrases, no -ing).